python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -673,6 +688,7 @@ def construct_dagster_k8s_job(
                 "restart_policy": "Never",
                 "containers": [container_config] + user_defined_containers,
                 "volumes": volumes,
+                "scheduler_name": scheduler_name,
```

Comment:
I assume nothing bad/different happens when this is set to None vs. the key being missing entirely? In both cases it falls back to the default? lots of stuff would break if it iddn't

Reply:
yeah- I think it's None by default. I suppose to be extra safe we can put it behind a merge dict